pkgs/development/python-modules/pyside6/default.nix

```
@@ -1,29 +1,18 @@
 { lib
 , stdenv
-, fetchpatch2
 , cmake
 , ninja
 , python
 , moveBuildTree
 , shiboken6
-, libxcrypt
 }:
 
 stdenv.mkDerivation rec {
   pname = "pyside6";
 
   inherit (shiboken6) version src;
 
-  sourceRoot = "pyside-setup-everywhere-src-${version}/sources/${pname}";
-
-  patches = [
-    # Needed to build against qt 6.5.3, until pyside 6.5.3 is released
-    (fetchpatch2 {
-      url = "https://code.qt.io/cgit/pyside/pyside-setup.git/patch/sources/pyside6?id=63ef7628091c8827e3d0d688220d3ae165587eb2";
-      hash = "sha256-TN1xdBkrzZhNontShMC1SKyJK6a8fOk/Di3zX3kv5+I=";
-      stripLen = 2;
-    })
-  ];
+  sourceRoot = "pyside-setup-everywhere-src-${lib.versions.majorMinor version}/sources/${pname}";
```

```suggestion
  sourceRoot = "pyside-setup-everywhere-src-${lib.removeSuffix ".0" version}/sources/${pname}";
```

Rationale: since 6.5.2 used the full version, but 6.6.0 only uses 6.6, it seems stripping the .0 instead of using majorMinor is the more long-lasting option here

---

That's a fabulous idea, thank you!